### PR TITLE
removes timestamp from log message

### PR DESCRIPTION
**Pull request by favadi**

### CODE REVIEW

File: log.go [hunk 1]
```diff
@@ -50,7 +64,10 @@ func NewEventHook(source string, levels []logrus.Level) (*EventHook, error) {
 
 // Fire extracts logrus entry and sends to window event log
 func (hook *EventHook) Fire(entry *logrus.Entry) error {
-	msg, err := entry.String()
+	msg, err := formatEntry(entry)
+	if err != nil {
+		return err
```
Comment (meomap) [inline log.go]: please wrap errors

File: log.go [hunk 2]
```diff
@@ -29,6 +29,20 @@ func isErrRegistryKeyExist(err error) bool {
...
+// formatEntry formats the logrus entry to a message without redundant
+// information (timestamp, log level) to send to windows log.
+func formatEntry(entry *logrus.Entry) (string, error) {
+	formatter := &logrus.TextFormatter{
+		DisableColors: true,
+		DisableTimestamp: true,
+	}
+	serialized, err := formatter.Format(entry)
+	if err != nil {
+		return "", err
```
Comment (meomap) [inline log.go]: please wrap error here also

**Review by meomap — COMMENTED** [review]
only minor things about error wrap

File: log.go (hunk 1 above)
Comment (favadi) [reply log.go]: fixed in 1a0409b69966ed0fdb1af43e1bd8751bae81506e.

File: log.go (hunk 2 above)
Comment (favadi) [reply log.go]: fixed in 1a0409b69966ed0fdb1af43e1bd8751bae81506e.

**Pull request merged**